src/main/java/org/folio/dew/batch/acquisitions/edifact/config/ExportConfig.java

```diff
@@ -0,0 +1,44 @@
+package org.folio.dew.batch.acquisitions.edifact.config;
+
+import org.folio.dew.batch.acquisitions.edifact.mapper.CsvMapper;
+import org.folio.dew.batch.acquisitions.edifact.mapper.ExportResourceMapper;
+import org.folio.dew.batch.acquisitions.edifact.mapper.converter.CompOrderEdiConverter;
+import org.folio.dew.batch.acquisitions.edifact.mapper.converter.CompPoLineEdiConverter;
+import org.folio.dew.batch.acquisitions.edifact.mapper.EdifactMapper;
+import org.folio.dew.batch.acquisitions.edifact.services.ConfigurationService;
+import org.folio.dew.batch.acquisitions.edifact.services.ExpenseClassService;
+import org.folio.dew.batch.acquisitions.edifact.services.HoldingService;
+import org.folio.dew.batch.acquisitions.edifact.services.IdentifierTypeService;
+import org.folio.dew.batch.acquisitions.edifact.services.LocationService;
+import org.folio.dew.batch.acquisitions.edifact.services.MaterialTypeService;
+import org.folio.dew.batch.acquisitions.edifact.services.OrdersService;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan({ "org.folio.dew.batch.acquisitions.edifact" })
+public class ExportConfig {
```

Comment:
this module is used for other export types as well, so can name this class as AcquisitionExportConfig for example